src/main/java/org/codejive/properties/Properties.java

```diff
@@ -2,12 +2,36 @@
 
 import static org.codejive.properties.PropertiesParser.unescape;
 
-import java.io.*;
```

Comment:
Can we leave the imports as-is please? :-)

Reply:
The build failed with it, so I ran `mvn spotless:apply` (as it asked me to do so) and it performed this change